services/scraper/search_keyword_test.go

```diff
@@ -18,16 +21,71 @@ var _ = Describe("Scraper/SearchKeyword", func() {
 		cassetteName := "scraper/success_valid_params"
 
 		RecordCassette(cassetteName, visitURL)
+		PreparePositionTable()
```

Comment:
How about naming it:

```suggestion
		SeedPositionTable()
```